recipes/pyabel/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "PyAbel" %}
+{% set version = "0.7.4" %}
+{% set md5 = "1f9be2ecdc1ea8424000d62e2d6a4b10" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  md5: {{ md5 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - toolchain
+    - cython
+    - numpy >=1.6
+    - setuptools >=16.0
+    - scipy >=0.14
+    - six >=1.10.0
```

Comment:
Are all of these needed at build time or only at install time? Also is this linking against `cython`?